data/assets/examples/rotation/fixedrotation/fixed_axis-mapping.asset

```diff
@@ -0,0 +1,32 @@
+-- Axis Mapping
+-- This asset creates a rotation that shows coordinate axes in which the x and the y axes
+-- are flipped. While this could also be achieved with a ConstantRotation class, this
```

Comment:
```suggestion
-- are flipped. While this could also be achieved with a `ConstantRotation` class, this
```

or 

```suggestion
-- are flipped. While this could also be achieved with a [ConstantRotation](#base_transform_rotation_constant) class, this
```